contractcourt/chain_arbitrator.go

```diff
@@ -623,6 +623,11 @@ func (c *ChainArbitrator) ForceCloseContract(chanPoint wire.OutPoint) (*wire.Msg
 
 	log.Infof("Attempting to force close ChannelPoint(%v)", chanPoint)
 
+	if err := c.cfg.DisableChannel(chanPoint); err != nil {
```

Comment:
If we fail to force close the channel below, then we'll leave the channel disabled. I guess this is fine since in most force close cases the peer is offline, so it'll be re-enabled once the peer reconnects.

Reply:
yep exactly